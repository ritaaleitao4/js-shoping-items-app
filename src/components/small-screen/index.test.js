import SmallScreen from '.';

describe('footer component', () => {
    it('it renders correctly', () => {
        expect(SmallScreen()).toMatchSnapshot();
    });
});