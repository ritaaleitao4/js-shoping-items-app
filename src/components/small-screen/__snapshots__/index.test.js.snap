// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component it renders correctly 1`] = `
<section class="small-screen">
  I look much better in a wide window ;)
</section>
`;
